android/app/proguard-rules.pro

```diff
@@ -9,13 +9,6 @@
 
 # Add any project specific keep options here:
 
-# If your project uses WebView with JS, uncomment the following
-# and specify the fully qualified class name to the JavaScript interface
-# class:
-#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
-#   public *;
-#}
-
```

Comment:
Technically, this should happen after the upgrade commit because RN
    v0.60.0 newly omits WebView in RN core. But we're not using the
    WebView in RN core, we're using `react-native-webview`, from
    `react-native-community`. So the absence of this hint will be fine.

Also it will be fine because it's a hint we've been ignoring all this time anyway :wink: 

(I don't even really understand what it's trying to say.)